src/sqswatcher/plugins/slurm.py

```diff
@@ -48,7 +49,7 @@ def _restart_master_node():
 
 def _restart_multiple_compute_nodes(hostnames, cluster_user):
     command = (
-        "if [ -f /etc/systemd/system/slurmd.service ]; "
+        "if [ -e /usr/lib/systemd ]; "
```

Comment:
I think Ubuntu puts systemd in /lib. This would have to cope with both locations. Similar problems in other areas.

Reply:
Good to know. I guess both locations could be checked.